app/controllers/fe/concerns/answer_pages_controller_concern.rb

```diff
@@ -45,7 +45,12 @@ def update
       params[:reference].keys.each do |id|
         reference_params = params.fetch(:reference)[id].permit(:relationship, :title, :first_name, :last_name, :phone, :email, :is_staff)
 
+        # The call to Application#reference_sheets is supposed to update the visibility cache and return the right references, but
+        # currently it does not do that right.  I have to call @answer_sheet.reference_sheets manually once, then it gets the right
+        # list.  I think this may be caused from a change in rails 5
+        @answer_sheet.reference_sheets
```

Comment:
Would `@answer_sheet.reload` work?

Reply:
Perhaps.. I'll try it

Reply:
Yes, .reload works.  Do you think I should keep the comment explanation?

Reply:
Yes